### PR TITLE
Pop up menu and adding html dynamically through JavaScript

### PR DESCRIPTION
**Pull request by keebot-png**
The Following actions were taken to complete this project:

Ran and passed all linter errors.

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

When the user clicks (or taps) the close (X) button, the popup disappears.

In order to associate each project with the popup window details, all of the information for all projects were stored in an array using objects for each unique project, this includes at a minimum: name, description, featured image, technologies, link to live version, link to source.

You need to update the main page so that the projects section is created dynamically using the information stored in that
JavaScript object.

The last step was to style and implement the pop up window.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -21,4 +21,187 @@ function closeScreen() {
...
+            </div>
+            
+            <div class="modal-bottom">
+                <p>${popData[data].Description}</p>
+            </div>
+            <div class="buttons-1">
+                <button class="modal-11">See live <img src="${popData[data].liveIcon}" alt="live logo icon"></button>
+                <button class="modal-12">See Source <img src="${popData[data].githubIcon}" alt="github logo"></button>
+            </div>
+        </div>
```
Comment (Abenezer-Tilahun) [inline script.js]: - [ ] Kindly work on Adding event listeners to the link to live version, and link to the project source code. When I click on the buttons nothing happens.

File: script.js [hunk 2]
```diff
@@ -21,4 +21,187 @@ function closeScreen() {
...
+      </div>
+      <div class="works-cta">
+          <button type="submit" class="project-button" onclick="modalSection(${i})">
+              See Project
+          </button>
+      </div>
+  </div>
+</div>
+  `;
+}
```
Comment (Abenezer-Tilahun) [inline script.js]: - [ ] Kindly follow  design for  mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). Please try to adjust the alignment based on the Figma design. Minimum mobile screen size should be 375px.
The Figma design Look likes: 
cap1
Capture

Your design at Minimum mobile screen size ( 375px) Look likes this : 
Cap2

The Pop up is not fully responsive at the bottom "See Live " and "See Source code " is not visible. Kindly adjust it.

Captur1

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @keebot-png  

Good job so far 👏!
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README 

almost

## Required Changes ♻️
_Check the comments below the review._

- [ ] For this project you are required to use The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters configuration, Kindly update it , still In your  project you configure HTML and CSS  linter configuration. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by keebot-png** [comment]
Hi @Abenezer-Tilahun I have completed the changes with regards to the project link and the source link as requested. However, the display cards are fully responsive on my browser and may look a bit different from yours. I'm guessing you used Safari or Firefox, I kindly ask that you may excuse me from making the respective card changes as we are not obligated to check for browser compatibility. As for the linters, all of them passed and were successful including the JavaScript linters, I'm not sure why you said I have not updated my linters configuration because they all passed the checks. I used Chrome as my default browser.

**Comment by Abenezer-Tilahun** [comment]
Hello @keebot-png Yeah You are right , for making your website responsive and compatibility with any browser that why i recommend you to adjust it. But if it's okay with you, You can ask for Re-review for project.

**Comment by keebot-png** [comment]
> Hello @keebot-png Yeah You are right , for making your website responsive and compatibility with any browser that why i recommend you to adjust it. But if it's okay with you, You can ask for Re-review for project.

@Abenezer-Tilahun Ive made the respective changes including the linters.yml file. Thanks for the professional review

File: script.js [hunk 3]
```diff
@@ -21,4 +21,187 @@ function closeScreen() {
...
+const popData = [
+  {
+    classNO: 'n1',
+    Subtitle: 'Keeping track of \nhundreds of components',
+    Technologies: ['Ruby on Rails', 'CSS', 'Javascript', 'HTML'],
+    Description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry&#39s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+    githubIcon: './images/github-log.svg',
+    liveIcon: './images/live-icon.svg',
+    ProjectLink: 'https://keebot-png.github.io/personal_portfolio/',
+    GitHubLink: 'https://github.com/keebot-png/personal_portfolio',
```
Comment (Ismailco) [inline script.js:36]: - [ ] [OPTIONAL] The logos for live link and source are not displayed properly kindly consider fixing this so the icons are displayed correctly

Screenshot_20220916_172709

**Review by Ismailco — APPROVED** [review]
### Hi @keebot-png, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**